lib/scryptenc/scryptenc.h

```diff
@@ -62,23 +62,22 @@
  * compared to the computed limits and an error is returned if decrypting
  * the data would take too much memory or CPU time.
  */
-/**
- * Return codes from scrypt(enc|dec)_(buf|file):
- * 0	success
- * 1	getrlimit or sysctl(hw.usermem) failed
- * 2	clock_getres or clock_gettime failed
- * 3	error computing derived key
- * 4	could not read salt from /dev/urandom
- * 5	error in OpenSSL
- * 6	malloc failed
- * 7	data is not a valid scrypt-encrypted block
- * 8	unrecognized scrypt format
- * 9	decrypting file would take too much memory
- * 10	decrypting file would take too long
- * 11	password is incorrect
- * 12	error writing output file
- * 13	error reading input file
- */
+
+/* Return codes from scrypt(enc|dec)_(buf|file|prep). */
+#define SCRYPT_OK	0	/* success */
+#define SCRYPT_ELIMIT	1	/* getrlimit or sysctrl(hw.usermem) failed */
+#define SCRYPT_ECLOCK	2	/* clock_getres or clock_gettime failed */
+#define SCRYPT_EKEY	3	/* error computing derived key */
+#define SCRYPT_ESALT	4	/* could not read salt from /dev/urandom */
```

Comment:
Can you drop the "from /dev/urandom" from here?  Theoretically we might use a different entropy backend in the future.